FrontEnd/src/components/RestorePassword/components/restorepassword/restorepassword-form/EmailFormContent.js

```diff
@@ -0,0 +1,102 @@
+import { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import axios from 'axios';
+import styles from './EmailFormContent.module.css';
+
+export function SendEmailRestorePasswordFormContentComponent({ setIsValid }) {
+  const navigate = useNavigate();
+
+  const errorMessageTemplates = {
+    required: 'Обов’язкове поле',
+    email: 'Email не відповідає вимогам',
+  };
+
+  const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
```

Comment:
Do we use `emailPattern` only here? It would be good to create a constant for it and reuse it in all places (Login, Register...)

Reply:
Fixed